integration-tests/tracecontext-integration-test.sh

```diff
@@ -0,0 +1,26 @@
+
+#!/bin/sh
+set -e
+# hard-coding the git tag to ensure stable builds.
+TRACECONTEXT_GIT_TAG="98f210efd89c63593dce90e2bae0a1bdcb986f51"
+# clone w3c tracecontext tests
+mkdir -p target
+rm -rf ./target/trace-context
+git clone https://github.com/w3c/trace-context ./target/trace-context
+cd ./target/trace-context && git checkout $TRACECONTEXT_GIT_TAG && cd -
+pip3 install setuptools;
+pip3 install aiohttp; 
+node ./integration-tests/propagation-validation-server/validation-server.js 1>&2 &
+EXAMPLE_SERVER_PID=$!
+# give the app server a little time to start up. Not adding some sort
+# of delay would cause many of the tracecontext tests to fail being
+# unable to connect.
+sleep 1
```

Comment:
if we find this gets flaky in the future, a health-check approach can be considered. works fine for me, currently.

Reply:
Fasho, sounds good